### PR TITLE
Renamed secondaryLabel to optionSelectLabel and is now always displayed

### PR DESCRIPTION
**Pull request by chrisekelley**

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -61,31 +61,31 @@
...
-        <template>
-          <tangy-form id="my-form" title="My Form">
-            <tangy-form-item id="item1">
-              <tangy-input name="input1" label="What is your first name?"required></tangy-input>
-            </tangy-form-item>
-            <tangy-form-item id="item2">
-              <tangy-input name="input2" label="What is your last name?" required></tangy-input>
-            </tangy-form-item>
-          </tangy-form>
+<!--      <h3>Basic tangy-form demo</h3>-->
```
Comment (chrisekelley) [inline demo/index.html]: Commented out this first demo; it prevents the viewing of the subsequent demos.

**Comment by chrisekelley** [comment]
Fixes https://github.com/Tangerine-Community/Tangerine/issues/1602

**Comment by chrisekelley** [comment]
Resolves https://github.com/Tangerine-Community/Tangerine/issues/1602

**Comment by rjsteinert** [comment]
Use of that `mdc-error` class looks good. In regard to the breaking API change of `secondaryLabel` to `optionSelectLabel`, it looks like we [never fully baked support](https://github.com/Tangerine-Community/tangy-form-editor/commits/v6.0.0/widget/tangy-select-widget.js) for modifying `secondaryLabel` in `tangy-form-editor` so there are probably few uses of it. However we'll want to make sure to mention this in the CHANGELOG for both `tangy-form` and `tangerine`.  The other option is we could use the `secondaryLabel` name for that attribute, or both :).

Let's also check into why that first demo is breaking. If that is breaking then we have something fishy going on. Lastly, Travis is reporting the tangy-select tests are failing and need looking into.

**Comment by chrisekelley** [comment]
I just added a PR to remove secondaryLabel from t-f-editor: https://github.com/Tangerine-Community/tangy-form-editor/pull/67

**Comment by rjsteinert** [comment]
@chrisekelley Heads up, tests are still failing due to the changes in this PR.

Screen Shot 2019-08-23 at 11 34 50 AM

**Pull request merged**